orchagent/switchorch.cpp

```diff
@@ -1045,6 +1274,10 @@ void SwitchOrch::doTask(Consumer &consumer)
     {
         doCfgSwitchHashTableTask(consumer);
     }
+    else if (tableName == CFG_SWITCH_TRIMMING_TABLE_NAME)
+    {
+        doCfgSwitchTrimmingTableTask(consumer);
```

Comment:
@nazariig Could you create sonic-buildimage PR with the trim global config and enable trim on the queues?

Reply:
@kperumalbfn discussed offline. Considering as resolved